person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name
+  attr_reader :id, :parent_permission
+  attr_writer :age
```

Comment:
- [ ] the `age` should be added to the attr_accessor instead of the attr_writer.